gallery/how_to/use_tvms_python_api/tvmc_python.py

```diff
@@ -0,0 +1,236 @@
+# target machine can understand. 
+# 
+# In order to compile a model a tvm.target string is required. 
+# To learn more about tvm.targets and their options look at the `documentation <https://tvm.apache.org/docs/api/python/target.html>`_. 
+# Some examples include:
+# 1. cuda (nvidia gpu)
+# 2. llvm (cpu)
+# 3. llvm -mcpu=cascadelake (intel cpu)
+#
+#  .. code-block:: python
+#      package = tvmc.compile(model, target="llvm") #Step 2: Compile
+# 
+# The compilation step returns a package.
+# 
+
+################################################################################
+# Step 3: Run
+# -----------
+# The compiled package can now be run on the hardware target. The device 
+# input options are: cpu, cuda, cl, metal, and vulkan.
+# 
+#  .. code-block:: python
+#      result = tvmc.run(package, device="cpu") #Step 3: Run
+# 
+# And you can print the results:
+# ``print(results)``
+# 
+
+################################################################################
+# Step 1.5: Tune [Optional & Recommended]
+# ---------------------------------------
+# Run speed can further be improved by tuning. This optional step uses 
+# machine learning to look at each operation within a model (a function) and 
+# tries to find a faster way to run it. We do this through a cost model, and 
+# bench marking possible schedules.
+# 
+# The target is the same as compile. 
+# 
+#  .. code-block:: python
+#      tvmc.tune(model, target="llvm") #Step 1.5: Optional Tune
+# 
+# The terminal output should look like: 
+# [Task  1/13]  Current/Best:   82.00/ 106.29 GFLOPS | Progress: (48/769) | 18.56 s
+# [Task  1/13]  Current/Best:   54.47/ 113.50 GFLOPS | Progress: (240/769) | 85.36 s
+# .....
+# 
+# There may be UserWarnings that can be ignored. 
+# This should make the end result faster, but it can take hours to tune. 
+# 
+
+################################################################################
+# Save and then start the process in the terminal:
+# ------------------------------------------------
+# 
+#  .. code-block:: python
+#      python my_tvmc_script.py
+# 
+# Note: Your fans may become very active
+# 
+
+################################################################################
+# Example results:
+# ----------------
+# 
+#   .. code-block:: python
+#      Time elapsed for training: 18.99 s
+#      Execution time summary:
+#      mean (ms)   max (ms)   min (ms)   std (ms) 
+#        25.24      26.12      24.89       0.38 
+#      
+#      Output Names:
+#       ['output_0']
+#
+
+Additional TVMC Functionalities
+===============================
+
+################################################################################
+# Saving the model
+# ----------------
+# 
+# To make things faster for later, after loading the model (Step 1) save the relay version. 
+# 
+#   .. code-block:: python
+#      model = tvmc.load('my_model.onnx') #Step 1: Load
+#      model.save(model_path) 
+# 
```

Comment:
It would be good to have some information on what file you actually get, as the result of `model.save`.